tao_compiler/mlir/xla/ral/BUILD

```diff
@@ -477,22 +477,65 @@ tf_gpu_kernel_library(
 ###########################################################################################
 # The targets following are for tf bridge side build. Get rid of all the deps from tf source code, only use some of the `.bzl` utils from tensorflow.
 
+cc_library(
+    name = "disc_mkl_bridge",
+    hdrs = glob([
+        "context/mkldnn/ideep/*.hpp",
+        "context/mkldnn/ideep/ideep/*.hpp",
+        "context/mkldnn/ideep/ideep/operators/*.hpp",
+    ]),
+    linkopts = [
+        "-L$$(dirname $(locations @onednn//:onednn_lib)",
+        "-Wl,--start-group",
+        "$(location @onednn//:onednn_lib)",
+        "-Wl,--end-group",
+    ] + if_disc_x86([
+        "-L$(location @mkl_static//:lib)/",
+        "-Wl,--start-group",
+        "$(location @mkl_static//:lib)/libmkl_intel_ilp64.a",
+        "$(location @mkl_static//:lib)/libmkl_gnu_thread.a",
+        "$(location @mkl_static//:lib)/libmkl_core.a",
+        "-Wl,--end-group",
+    ]) + if_disc_aarch64([
+        "-L$$(dirname $(location @acl_compute_library//:build/libarm_compute.a)",
+        "-Wl,--start-group",
+        "$(location @acl_compute_library//:build/libarm_compute.a)",
```

Comment:
Is it OK to explicitly depend on `@acl_compute_library//:acl` instead of writing such `-L`s and `.a`s? An example: https://github.com/bazelbuild/rules_foreign_cc/blob/main/examples/make_simple/BUILD.bazel#L32

Reply:
Here we try to get the location of static libraries from `@acl_compute_library`. However the target `@acl_compute_library//:acl` has 5 outputs as below, getting location will fail since bazel doesn't know which target's location to get. The example from `make_simple` works because it is used as `deps`, here we need the specific location.
`bazel-bin/external/acl_compute_library/acl
bazel-bin/external/acl_compute_library/acl/build/libarm_compute.a
bazel-bin/external/acl_compute_library/acl/build/libarm_compute_core.a
bazel-bin/external/acl_compute_library/acl/build/libarm_compute_graph.a
bazel-bin/external/acl_compute_library/copy_acl
`

Reply:
sorry, i miss the `-Wl` flag..